pages/dashboard/index.js

```diff
@@ -1,6 +1,10 @@
 const { page } = require('@asl/service/ui');
 const taskList = require('@asl/pages/pages/task/list/router');
 
+function userIsNtcoAtEst(profile, estId) {
+  return (profile.roles.find(role => role.establishmentId === estId) || {}).type === 'ntco';
```

Comment:
```suggestion
  return !!profile.roles.find(role => role.establishmentId === estId && role.type === 'ntco');
```

Otherwise this will return false if the user has another non-ntco role at the same establishment (and that appears before ntco in the roles list)

Reply:
how to even computer